cmd/repo-operations.go

```diff
@@ -366,7 +366,7 @@ func openPullRequest(config *GitXargsConfig, repo *github.Repository, branch str
 	newPR := &github.NewPullRequest{
 		Title:               github.String(titleToUse),
 		Head:                github.String(branch),
-		Base:                github.String("master"),
+		Base:                github.String(repo.GetDefaultBranch()),
```

Comment:
Since `repo.GetDefaultBranch()` is being called twice, you can probably define a constant above:

```go
const repoDefaultBranch = repo.GetDefaultBranch()
```